java-client/src/main/java/org/opensearch/client/transport/aws/AwsSdk2Transport.java

```diff
@@ -96,20 +96,42 @@ public AwsSdk2Transport(
             @Nonnull String host,
             @Nonnull Region signingRegion,
             @CheckForNull AwsSdk2TransportOptions options) {
-        this(httpClient, null, host, signingRegion, options);
+        this(httpClient, null, host, "es", signingRegion, options);
```

Comment:
```suggestion
        this(httpClient, null, host, "os", signingRegion, options);
```

Reply:
Seem like es -> elasticsearch? :)

Reply:
Oh sorry, I see it now - the value is coming from AWS

Reply:
I think I'd prefer it to be a string because I am not so sure that list is finite. I'm trying to find out whether the signing service name can be other than `es` and `aoss` in the case of a proxy. Can we leave it as is for now?